tripal_biomaterial/includes/TripalFields/sep__biological_sample/sep__biological_sample.inc

```diff
@@ -119,102 +207,140 @@ class sep__biological_sample extends ChadoField{
    * @throws \Exception
    */
   public function load($entity) {
-    if ($entity->chado_table == 'analysis') {
-      $this->analysisCount($entity);
-    }
+    $record = $entity->chado_record;
 
-    if ($entity->chado_table == 'project') {
-      $this->projectCount($entity);
-    }
+    $field_name = $this->field['field_name'];
+    $field_type = $this->field['type'];
+    $field_table = $this->instance['settings']['chado_table'];
+    $field_column = $this->instance['settings']['chado_column'];
+    $base_table = $this->instance['settings']['base_table'];
+
+    $name_term = chado_get_semweb_term('biomaterial', 'name');
+    $description_term = chado_get_semweb_term('biomaterial', 'description');
+    // Use fixed terms for these values which derive from the biomaterialprop table,
+    // type_id + value columns. This module's installer loads biomaterial_property cv
+    // and links it to tripal db. tripal_eutils module uses ncbi_properties cv linked
+    // to ncbi_properties db. Static term here allows this field to work with both.
+    $tissue_term = 'tripal:tissue';
+    $treatment_term = 'tripal:treatment';
 
```

Comment:
Two issues here:
1.  I don't think we should be hard-coding some attributes to appear for the biomaterial over others. Not all NCBI biomaterials have the tissue and treatment and there's nothing in this module that forces the user to provide these attributes for every biomaterial.  I think these were here from before but I don't know why.   I think we should think about removing them.  
2. Also, we shouldn't be using vocabulary terms from a `tripal` vocabulary because tripal doesn't have a vocabulary. They should come from a 'local' vocabulary, established by the site admins, or from a community derived vocabulary.  I believe I fixed all of these in this PR, https://github.com/tripal/tripal_analysis_expression/pull/392, so no need to do anything about it here. Just pointing out.